doc/source/async_api.rst

```diff
@@ -1,78 +1,94 @@
 Async API (Experimental)
 ========================
 
-Since Python 3.5, it is possible to write concurrent code using the ``async/await`` `syntax <https://docs.python.org/3/library/asyncio.html>`__.
+Since Python 3.5, it is possible to write concurrent code using the
+``async/await`` `syntax <https://docs.python.org/3/library/asyncio.html>`__.
 
-This document describes Ray's support for asyncio, which enables integration with popular async frameworks (e.g., aiohttp, aioredis, etc.) for high performance web and prediction serving.
+This document describes Ray's support for asyncio, which enables integration
+with popular async frameworks (e.g., aiohttp, aioredis, etc.) for highly
+concurrent workloads.
```

Comment:
can you provide a quickstart

Reply:
if the below is a quick start, please remove the header line

Reply:
added quickstart